packages/commons/src/fhir/extension.ts

```diff
@@ -272,3 +274,20 @@ export function findExtension<URL extends AllExtensions['url']>(
       extension.url === url
   )
 }
+
+export type URNReference = `urn:uuid:${UUID}`
```

Comment:
This is already defined here: https://github.com/opencrvs/opencrvs-core/blob/release-7035/packages/commons/src/fhir/index.ts#L39